criu/files-reg.c

```diff
@@ -826,20 +827,22 @@ static int dump_ghost_file(int _fd, u32 id, const struct stat *st, dev_t phys_de
 		fd = open(lpath, O_RDONLY);
 		if (fd < 0) {
 			pr_perror("Can't open ghost original file");
-			return -1;
+			goto err_out;
 		}
 
 		if (gfe.chunks)
-			ret = copy_file_to_chunks(fd, img, st->st_size);
+			exit_code = copy_file_to_chunks(fd, img, st->st_size);
 		else
-			ret = copy_file(fd, img_raw_fd(img), st->st_size);
+			exit_code = copy_file(fd, img_raw_fd(img), st->st_size);
```

Comment:
The whole idea of exit_code is not to use it as a return value for other functions

Reply:
ping...

Reply:
I hope I got it finally right.